server/app/services/customer_db_service.py

```
@@ -0,0 +1,430 @@
+        await db.commit()
+        
+        logger.info(f"Created new customer record: {phone_number} for org {organization_id}")
+        return new_customer
+    
+    @staticmethod
+    async def send_pre_call_sms(
+        db: AsyncSession,
+        customer_db: CustomerDB,
+        call_reason: str,
+        webhook_base_url: str,
+        time_slot: str = None
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)